portfolio/src/components/Connect.vue

```diff
@@ -1,105 +1,105 @@
-        >
-            {{alertMessage}}
-        </v-alert>
-        <h1 style="width: 100%; margin-top: 1rem;">Connect</h1>
-        <div id="social-media-container">            
-            <a class="media-box" href="https://github.com/NicolasCapparelli">
-                <img src="/assets/GitHub.svg" alt="github" >
-            </a>
-    
-            <a class="media-box" style="display: flex; align-items: center; justify-content: center;" href="https://linkedin.com/in/NicolasCapparelli">
-                <img style="width: 80%;" src="/assets/linkedin-logo-white.svg" alt="linkedin">
-            </a>
-    
-            <a class="media-box" href="mailto:capparellinicog@hotmail.com">
-                <img style="width: 100%" src="/assets/email-white.svg" alt="email">
-            </a>
-        </div>
+  <div class="page-container" id="connect-container">
+      
+    <v-alert 
+      id="connect-alert"
+      v-model="alert"
+      :type="alertType"
+      dismissible
+      elevation="3"
+      transition="scroll-y-transition"
+    >
+      {{alertMessage}}
+    </v-alert>
+
+    <h1 style="width: 100%; margin-top: 1rem;">Connect</h1>
+    <div id="social-media-container">            
+      <a class="media-box" href="https://github.com/NicolasCapparelli">
+        <img src="/assets/GitHub.svg" alt="github" >
+      </a>
 
-        <h1 style="width: 100%;">Leave a Comment</h1>
-        <CommentList :alertDelegate="alertDelegate"/>        
+      <a class="media-box" style="display: flex; align-items: center; justify-content: center;" href="https://linkedin.com/in/NicolasCapparelli">
+        <img style="width: 80%;" src="/assets/linkedin-logo-white.svg" alt="linkedin">
+      </a>
 
+      <a class="media-box" href="mailto:capparellinicog@hotmail.com">
+        <img style="width: 100%" src="/assets/email-white.svg" alt="email">
+      </a>
     </div>
+
+    <h1 style="width: 100%;">Leave a Comment</h1>
+    <CommentList :alertDelegate="alertDelegate"/>        
+
+  </div>
 </template>
 
 <script>
-import CommentList from './sub-components/CommentList'
+import CommentList from './sub-components/CommentList.vue';
 
 export default {
-    name: 'Connect',
-    
-    data () {
-        return {
-            alertMessage: "",
-            alertType: "success",
-            alert: false,
-        }
-    },
-
-    components: {
-        CommentList
-    },
-
-    methods: {
-        alertDelegate: function (isVisibile, type, message) {
-            this.alert = isVisibile
-            this.alertType = type
-            this.alertMessage = message
-        }
+  name: 'Connect',
+  
+  data () {
+    return {
+      alertMessage: "",
+      alertType: "success",
+      alert: false,
+    }
+  },
+
+  components: {
+    CommentList
+  },
+
+  methods: {
+    alertDelegate: function (isVisibile, type, message) {
+      this.alert = isVisibile;
+      this.alertType = type;
+      this.alertMessage = message;
     }
+  }
```

Comment:
what is this for

Reply:
`alertDelegate` is passed as a prop to `CommentList`, which is called when the user tried to delete a comment. If the delete was successful, `CommentList` will call `alertDelegate` with 

```
isVisible = true
alertType = "success"
message = "comment deleted successfully"
```

which will show a green snackbar component in the Connect page, vice versa for comment deletion unsuccessful